detector/aws_instance_invalid_ami.go

```diff
@@ -7,19 +7,30 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstanceInvalidAmi(issues *[]*issue.Issue) {
+type AwsInstanceInvalidAMIDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstanceInvalidAMIDetector() *AwsInstanceInvalidAMIDetector {
+	return &AwsInstanceInvalidAMIDetector{d}
+}
+
+func (d *AwsInstanceInvalidAMIDetector) Detect(issues *[]*issue.Issue) {
```

Comment:
__[gometalinter]__
* [golint] exported method AwsInstanceInvalidAMIDetector.Detect should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->